### ISSUE TITLE
Some questions about Sigmoid function

### ISSUE THREAD

**Issue by yueruchen** [xinntao/BasicSR#23]
Hi, thanks for your novel work, I have some questions.

In the Relativistic GAN the sigmoid function is used little differently from standard GAN. Specifically, I see in the paper it says the Sigmoid function is added after C(X_r) - C(X_f). But I haven't found the Sigmoid function in your code. Could you please help me to find the location and usage of it?

**Comment by xinntao** [comment]
Hi @yueruchen,
The loss for the discriminator is BCEWithLogitsLoss.
["This loss combines a Sigmoid layer and the BCELoss in one single class."](https://pytorch.org/docs/stable/nn.html?highlight=bcewithlogitsloss#torch.nn.BCEWithLogitsLoss)

So the sigmoid operation is in the loss function.

**Comment by yueruchen** [comment]
Cool! Thanks.
